toolkit/featureforms/src/main/java/com/arcgismaps/toolkit/featureforms/components/datetime/picker/DateTimePicker.kt

```diff
@@ -349,6 +355,33 @@ private fun DateTimePickerDialog(
 private object DateTimePickerDialogTokens {
     val containerHeight = 600.0.dp
     val containerWidth = 360.0.dp
+    val horizontalPadding = 25.dp
+}
+
+/**
+ * Constraints the width of the content based on the orientation and the [width]. If the
+ * current orientation is portrait, [Modifier.requiredWidth] used. If it is landscape then
+ * [Modifier.widthIn] is used. This is useful when different layouts are needed in portrait
+ * and landscape orientations.
+ */
+internal fun Modifier.widthWithOrientation(width: Dp) : Modifier = composed {
+    val configuration = LocalConfiguration.current
+    if (configuration.orientation == Configuration.ORIENTATION_LANDSCAPE) {
+        this.widthIn(width)
+    } else {
+        this.requiredWidth(width)
+    }
+}
+
+/**
+ * Scales the content appropriately if the current screen width is less than the [minWidth].
+ */
+internal fun Modifier.scaleIfNarrow(minWidth: Dp): Modifier = composed {
+    val screenWidth = LocalConfiguration.current.screenWidthDp.dp
+    val scale = if (screenWidth <= minWidth)
```

Comment:
slight nitpick. equality of floats is flaky anyway and we can avoid a floating point division this way.

```suggestion
    val scale = if (screenWidth < minWidth)
```